test/regressionTest.m

```diff
@@ -0,0 +1,57 @@
+function regressionTest(commitBefore,commitAfter)
```

Comment:
So is this intended for developers during hacking or as a check against develop? In the later case i would omit the second argument and call the first branchName